content_schemas/formats/shared/definitions/finder.jsonnet

```diff
@@ -288,6 +288,9 @@
       {
         type: "null",
       },
+      {
+        "$ref": "#/definitions/multiple_content_types",
```

Comment:
I'd suggest moving this into its own commit, and then doing the 'regenerate schemas' bit in a second commit. Makes it easier to see what you've changed vs the auto-generated changes that come later.